packages/atomic/src/components/atomic-refine-modal/atomic-refine-modal.pcss

```diff
@@ -11,3 +11,11 @@ select:hover + div,
 select:focus + div {
   @apply text-primary-light;
 }
+
+.centered {
+  @apply w-full max-w-lg mx-auto;
+}
+
+.adjust-for-scroll-bar {
+  margin-right: calc(-1 * (100vw - 3rem - 100%));
```

Comment:
Why is this needed, exactly? When I disable it locally on Chrome and Firefox, the scrollbar isn't affected.

Reply:
I think this would benefit from a comment/explanation about what it does and why it is needed.

It's a pretty arcane bit of CSS :D

Reply:
I created [a video](https://share.vidyard.com/watch/QFRrnPkuRd6UToirD91zDC?) where I toggle this line on and off. When off, the middle panel is offset from the header and footer by the width of the scroll bar.